environments/vm/secrets/vm.yml

```diff
@@ -26,6 +26,9 @@ engine_janus_secret: secret
 
 engine_ldap_password: secret
 
+# Minimum execution time in milliseconds when response is invalid
+engine_minimum_execution_time_on_invalid_response: 5000
```

Comment:
literally, this change :wink:

Why is it needed? The default is in `roles/engineblock/defaults/main.yml`, this is a VM specific override that is the exact same, what is the point of that?